packages/slidev/node/export.ts

```diff
@@ -400,8 +400,7 @@ export async function exportSlides({
     await genPagePng()
   }
   else if (format === 'md') {
-    await genPagePng()
-    await genPageMd(slides)
+    await Promise.all([ genPagePng(), genPageMd(slides)])
```

Comment:
The `genPageMd` function reads the `output` dir where the images are exported to then generate the md file, so we can't process them in parallel